packages/paste-core/components/editable-code-block/src/EditableCodeBlock.tsx

```diff
@@ -1,39 +1,168 @@
 import * as React from 'react';
 import {Box, type BoxProps} from '@twilio-paste/box';
+import {useTheme} from '@twilio-paste/theme';
 import {
   CodeEditor,
   CodeEditorPasteTheme,
   type CodeEditorProps,
   type Editor,
   type Monaco,
+  type IRange,
+  type IMarkdownString,
 } from '@twilio-paste/code-editor-library';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {Spinner} from '@twilio-paste/spinner';
+import {StylingGlobals} from '@twilio-paste/styling-library';
+
+const EditableCodeblockLoadingMessage: React.FC<{i18nLoadingLabel: string}> = ({i18nLoadingLabel}) => (
+  <>
+    <Spinner color="colorTextPrimaryWeak" decorative />
+    <ScreenReaderOnly>{i18nLoadingLabel}</ScreenReaderOnly>
+  </>
+);
+EditableCodeblockLoadingMessage.displayName = 'EditableCodeblockLoadingMessage';
+
+const InlineErrorStyles: {[key: string]: any} = {
```

Comment:
suggestion(non-blocking):

I'm not too fussed about actually building this in—it's more out of curiosity. Is it possible at all to get inline errors to show up as red blocks in the scrollbar, like the syntax errors do?

<img width="273" alt="Screenshot 2023-07-31 at 4 10 43 PM" src="https://github.com/twilio-labs/paste/assets/3821840/6dba38bc-f104-4b68-bbb8-46ea72929cf8">

Reply:
Done